boards/hail/.cargo/config.toml

```diff
@@ -0,0 +1,31 @@
+# Licensed under the Apache License, Version 2.0 or the MIT License.
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+# Copyright Tock Contributors 2024.
```

Comment:
Maybe a comment here noting that things are copied into this file so that hail can be the rust-stable test case board? Otherwise it's a bit unclear why this one is different.

Reply:
Done